docs/installation/configuration.md

```diff
@@ -185,3 +185,17 @@ This results in a server with:
 - Flannel backend set to `none`
 - The token set to `secret`
 - Debug logging enabled
+
+### Kubelet configuration
+
+:::info Version Gate
```

Comment:
I'd probably keep this short. No need to mention the bug, just mention which releases we support it on.